injects/sampler.js

```diff
@@ -40,8 +40,11 @@ function saveSample () {
 
 // start sampler on next tick, to avoid measuring the startup time
 process.nextTick(function () {
-  processStat.refresh()
-  scheduleSample()
+  // allow time delay to be specified before we start collecting data
+  setTimeout(() => {
```

Comment:
`process.nextTick` schedules work, and `setTimeout` does too. so we're waiting for a bit and then doing `setTimeout`, which also waits for a bit. that's unnecessary: we can only do the `setTimeout()` and it wuld have the same effect.

Reply:
@goto-bus-stop - Updated. Thanks.